runners/direct-java/src/main/java/org/apache/beam/runners/direct/GroupAlsoByWindowEvaluatorFactory.java

```diff
@@ -199,12 +196,10 @@ public void processElement(WindowedValue<KeyedWorkItem<K, V>> element) throws Ex
      * Returns an {@code Iterable<WindowedValue<InputT>>} that only contains non-late input
      * elements.
      */
-    public Iterable<WindowedValue<V>> dropExpiredWindows(
+    Iterable<WindowedValue<V>> dropExpiredWindows(
```

Comment:
Is there a reason why we duplicate `LateDataUtils.dropExpiredWindows`?

Reply:
Combination of ordering (I believe this was introduced first) plus a minor argument conflict (we use some fields here that don't have the same type as the `Utils` method (CounterCell vs Counter being the big difference)

Reply:
Thats unfortunate.